README.md

```diff
@@ -37,6 +37,10 @@ Timeplus Proton is a stream processing engine and database. It is a fast and lig
 ![Proton Architecture](design/proton-architecture.webp)
 See our [architecture](https://docs.timeplus.com/proton-architecture) doc for technical details and our [FAQ](https://docs.timeplus.com/proton-faq) for more information.
 
+## How is it different from ClickHouse?
```

Comment:
Thanks for adding this. LGTM. Please also update the links at the beginning of the README (between the badges and what is Timeplus Proton), so that people can jump directly to this section.